pkg/processor/runtime/python/py/test_wrapper.py

```diff
@@ -12,30 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import base64
 import functools
+import http.client
 import json
 import logging
 import operator
 import os
 import socket
+import socketserver
 import struct
 import sys
 import tempfile
 import threading
 import time
 import unittest.mock
-import http.client
-import socketserver
 
-import _nuclio_wrapper as wrapper
 import msgpack
 import nuclio_sdk
 import nuclio_sdk.helpers
+import pkg_resources
+import six
```

Comment:
it's no biggy, since it's just for testing (dev requirements) but you can do without six here, you just need a very simple subset of 1 simple method:
would save unnecessarily having the entire six package here just for this:
```
def ensure_str(self, s, encoding='utf-8', errors='strict'):

   # Optimization: Fast return for the common case.
    if type(s) is str:
        return s
    if isinstance(s, bytes):
        return s.decode(encoding, errors)
    raise TypeError(f"not expecting type '{type(s)}'")
```
Note I'm assuming here we're only having python3 by now.
If that won't be the case I wouldn't bother duplicating more intricate logic from six but revert back to importing it like you did